### PR TITLE
Rename the gem to `Repositorish`

### PR DESCRIPTION
**Pull request by ggcampinho**
- Rename the gem to `Repositorish`
- Rename the `ACTIVE_RECORD_NAMESPACE_REGEX` to `CHAINABLE_NAMESPACES_REGEX`, let the porpouse more clear

### CODE REVIEW

**Comment by shaneog** [comment]
You also need to change the name in the `.travis.yml` file.

File: lib/repositorish.rb [hunk 1]
```diff
@@ -20,8 +20,9 @@
 # UserRepository.confirmed
 # # => <User::ActiveRecord_Relation ...>
 # ```
-module Warehouse
-  ACTIVE_RECORD_NAMESPACE_REGEX = /(?:^|::)ActiveRecord(?:_(?:Association)?Relation)?(?:$|::)/.freeze
+module Repositorish
+  CHAINABLE_NAMESPACES = %w(ActiveRecord ActiveRecord_Relation ActiveRecord_AssociationRelation)
+  CHAINABLE_NAMESPACES_REGEX = /(?:^|::)(?:#{CHAINABLE_NAMESPACES.join('|')})(?:$|::)/.freeze
```
Comment (shaneog) [inline lib/repositorish.rb:25]: I think this is causing build failures: https://travis-ci.org/LoveMondays/repositorish/builds/90569762

File: .travis.yml [hunk 2]
```diff
@@ -19,5 +19,5 @@ deploy:
   gem: warehouse
```
Comment (shaneog) [inline .travis.yml]: Also change name here please!!

**Comment by vnegrisolo** [comment]
:+1:

**Comment by brennovich** [comment]
:+1:

**Comment by vnegrisolo** [comment]
:+1:

**Pull request merged**